lang/python/pillow/patches/0001-setup.py-use-patch-from-Debian-and-tweak-it.patch

```diff
@@ -0,0 +1,316 @@
+-            _add_directory(library_dirs, "/usr/local/lib")
+-            _add_directory(include_dirs, "/usr/local/include")
+-
+-            _add_directory(library_dirs, "/usr/lib")
+-            _add_directory(include_dirs, "/usr/include")
+-            # alpine, at least
+-            _add_directory(library_dirs, "/lib")
+-
+-        # on Windows, look for the OpenJPEG libraries in the location that
+-        # the official installer puts them
+-        if sys.platform == "win32":
+-            program_files = os.environ.get('ProgramFiles', '')
+-            best_version = (0, 0)
+-            best_path = None
+-            for name in os.listdir(program_files):
+-                if name.startswith('OpenJPEG '):
+-                    version = tuple(int(x) for x in
+-                                    name[9:].strip().split('.'))
+-                    if version > best_version:
+-                        best_version = version
+-                        best_path = os.path.join(program_files, name)
+-
+-            if best_path:
+-                _dbg('Adding %s to search list', best_path)
+-                _add_directory(library_dirs, os.path.join(best_path, 'lib'))
+-                _add_directory(include_dirs,
+-                               os.path.join(best_path, 'include'))
+ 
+         #
+         # insert new dirs *before* default libs, to avoid conflicts
+@@ -472,22 +404,10 @@ class pil_build_ext(build_ext):
+         feature = self.feature
+ 
+         if feature.want('zlib'):
+-            _dbg('Looking for zlib')
+-            if _find_include_file(self, "zlib.h"):
+-                if _find_library_file(self, "z"):
+-                    feature.zlib = "z"
+-                elif (sys.platform == "win32" and
+-                      _find_library_file(self, "zlib")):
+-                    feature.zlib = "zlib"  # alternative name
++            feature.zlib = "z"
+ 
+         if feature.want('jpeg'):
+-            _dbg('Looking for jpeg')
+-            if _find_include_file(self, "jpeglib.h"):
+-                if _find_library_file(self, "jpeg"):
+-                    feature.jpeg = "jpeg"
+-                elif (sys.platform == "win32" and
+-                      _find_library_file(self, "libjpeg")):
+-                    feature.jpeg = "libjpeg"  # alternative name
++            feature.jpeg = "jpeg"
+ 
+         feature.openjpeg_version = None
+         if feature.want('jpeg2000'):
+@@ -537,13 +457,7 @@ class pil_build_ext(build_ext):
+                     feature.imagequant = "libimagequant"
+ 
+         if feature.want('tiff'):
+-            _dbg('Looking for tiff')
+-            if _find_include_file(self, 'tiff.h'):
+-                if _find_library_file(self, "tiff"):
+-                    feature.tiff = "tiff"
+-                if (sys.platform in ["win32", "darwin"] and
+-                        _find_library_file(self, "libtiff")):
+-                    feature.tiff = "libtiff"
++            feature.tiff= "tiff"
```

Comment:
if you just add `feature.tiff= "tiff"` here and leave the removed lines, it has the same effect
makes patch smaller

Reply:
```suggestion
+            feature.tiff= "tiff" # openwrt: always use tiff
```
If we are to do this, I would suggest to add a comment--inline to keep your intention to keep the patch small--to indicate we're trumping the above code.  It should make it easier to understand what's going on.